### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by RicardoNeves**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,54 @@
...
+  ],
+  declarations: [
+    AppComponent
+  ],
+  bootstrap: [AppComponent]
+})
+export class AppModule { }
+```
+
+Follow the features documentation and add it on your html:
```
Comment (jmagalhaes-xing) [inline README.md]: I would remove "the features" and change "html" to "HTML"

File: README.md [hunk 2]
```diff
@@ -1,27 +1,54 @@
...
+```scss
+@import '~keds/scss/_theming';
+
+// Generate all styles for all components with one include
+@include keds-theme;
+```
+
+If you just want to use the styles from a specific component you can do it by having something like the following.
+```scss
+@import '~ngx-dawn/scss/_theming';
```
Comment (jmagalhaes-xing) [inline README.md]: ngx-dawn?

File: README.md [hunk 3]
```diff
@@ -1,27 +1,54 @@
...
+
+// Button styles only
+@include ngx-dawn-button-theme($palette);
```
Comment (jmagalhaes-xing) [inline README.md]: ngx-dawn?

File: README.md (hunk 2 above)
Comment (RicardoNeves) [reply README.md]: nice catch

**Pull request merged**